wasm/smolmix/Makefile

```
@@ -0,0 +1,35 @@
+build: build-rust
```

checkmake (0.3.2)

[warning] Target "build" should be declared PHONY.

(phonydeclared)